pdf viewer - annotations (#123)
DevExpress/dotnet-eud: interface-elements-for-desktop/articles/pdf-viewer/file-attachment.md

---
title: File Attachments
author: Alexander Temnov
legacyId: 115330
---
# File Attachments

> This document describes actions that can be performed on attachments in the PDF Viewer.

Reviewer: ```suggestion
This document describes the available actions for attachments in the PDF Viewer.
```

An attachment is a file that is attached to a PDF document. The PDF Viewer shows file attachments in the **Attachments** panel of the navigation pane.

![Attachment](../../images/img121136.png)

## Open an Attachment

> To open a file attachment for viewing, you need an application which supports the corresponding file format to be installed in your computer. To view an attachment, do one of the following.

Reviewer: ```suggestion
To open certain file attachments, you need an application that can read the corresponding file format. To view an attachment, do one of the following:
```
